e2e/_suites/fleet/fleet.go

```diff
@@ -307,15 +316,15 @@ func (fts *FleetTestSuite) anAgentIsDeployedToFleetWithInstallerAndFleetServer(i
 	containerName := fmt.Sprintf("%s_%s_%s_%d", profile, fts.Image+"-systemd", serviceName, 1) // name of the container
 
 	// enroll the agent with a new token
-	enrollmentKey, err := fts.kibanaClient.CreateEnrollmentAPIKey(fts.Policy)
+	enrollmentKey, err := fts.kibanaClient.CreateEnrollmentAPIKey(fts.FleetPolicy)
```

Comment:
Mmm, IIRC @blakerouse mentioned the token must be provided by the default policy, not the default fleet server policy.

Reply:
I think for the agent that is bootstrapping fleet server it was supposed to use the fleet server policy and all other agents use the default policy, @blakerouse is that correct?

Reply:
I verified with @EricDavisX that this should indeed point to the fleet server policy when bootstrapping an agent with fleet server

Reply:
Cool then. I took a note and added a comment in the src pointing to that. This should be changed then